doc/breaking_change_rules/1012.md

```diff
@@ -0,0 +1,32 @@
+### 1012 - Subgroup Removed
+
+**Description**: Checks whether an existing cmd is removed from the previous cli version. 
+
+**Analysis**: This is considered a breaking change because it may influence users' current workflow. This change requires to be released in breaking-change window.
```

Comment:
```suggestion
**Analysis**: This is considered a breaking change because it may influence users' current workflow. This change requires to be released during breaking-change window.
```